index.html

```diff
@@ -162,26 +162,38 @@ <h2>Introduction</h2>
   <p><a class="termref">Accessibility <abbr title="Application Programming Interfaces">APIs</abbr></a> currently support flat, unstructured strings for accessible names and descriptions.  The result of the name/description computation is thus a flat string. </p>
   <p>The terms "accessible name" and "accessible description" are used to emphasize that they are properties of <a class="termref">accessible objects</a> as exposed by <a class="termref">Accessibility <abbr title="Application Programming Interfaces">APIs</abbr></a>.  However, they are frequently referred to hereafter as simply "name" and "description".</p>
 </section>
+<section class="informative" id="terms">
+	<h1>Important Terms</h1>
+	<div>
+		<p>While some terms are defined in place, the following definitions are used throughout this document. </p>
+		<dl class="termlist">
+      <dt><dfn data-no>Tooltip attribute</dfn></dt>
```

Comment:
Terms are not exported by default, so can just be: 
```suggestion
      <dt><dfn>Tooltip attribute</dfn></dt>
```